.changeset/tricky-ducks-count.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': minor
+---
+
+Visual updates to Checkbox component
```

Comment:
Don't need the changeset as developers can't see this. Feel free to add `skip changelog` label.